lib/compiler-finder.ts

```diff
@@ -475,12 +475,13 @@ export class CompilerFinder {
     checkOrphanedProperties() {
         // Quickly check for any orphaned compilers
         let error = false;
+        const propsCheckLocal = this.ceProps('propsCheckLocal', true);
```

Comment:
I don't really like true here as a default

Reply:
Do we know when checking local properties would be useful? Who would set it to `true` ? (As the reason I asked for this is that I'd set it to `false` so I could check in code while I have local modifictations in local properties)